testing/gnome-builder/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=gnome-builder
+pkgver=3.30.3
```

Comment:
Yes, 3.32.x is out already, but that needs glib >= 2.60 which we don't have yet. Since glib 2.60 only offers meson as build system which produces broken static libs I'd rather not upgrade to it just yet.